server/repository/src/migrations/mod.rs

```diff
@@ -226,7 +226,9 @@ pub fn migrate(
     let final_database_version = get_database_version(connection);
 
     // Recreate views
-    if final_database_version >= min_version_for_dropping_views {
+    let last_version_in_migration_vec = migrations.last().unwrap().version();
+
+    if final_database_version >= last_version_in_migration_vec {
```

Comment:
This was added because re-creating views was failing as reason_options_id added in 2.8 was causing errors when database views were being recreating for versions 2.3 up until 2.8. This change was made as we hopefully don't need to re-create a views for a previous version with db schema from the latest version. If we do ever need to do that, then this will need a rethink.

Reply:
Awesome, I'd just add some comments here for clarity for people coming back!

```suggestion
     // .unwrap() is safe here, as we know the migrations vec is populated
    let latest_version = migrations.last().unwrap().version();

    // only want to apply views against the latest database schema
    if final_database_version >= latest_version {
```

Reply:
Updated here bac623e815a20dcafd07b4cbd3a13dda072a186f